.github/CHANGELOG.md

```diff
@@ -54,7 +55,8 @@
 
 This release contains contributions from:
 
-Thomas Bromley, Diego Guala, Anthony Hayes, Antal Száva
+Thomas Bromley, Diego Guala, Anthony Hayes, Antal Száva, Ryan Hill
```

Comment:
```suggestion
Thomas Bromley, Diego Guala, Anthony Hayes, Ryan Hill, Antal Száva
```

Reply:
Oops, we usually have 

> This release contains contributions from (in alphabetical order):

but looks like it wasn't added after the last release!